src/components/button/button.html

```diff
@@ -0,0 +1,3 @@
+<button type="button>">
```

Comment:
Each button element is per default from type button - except in forms (which we will get to later). Also, you had a typo in there 😉 
```suggestion
<button>
```